src/Stratis.Bitcoin/Configuration/Settings/ConnectionManagerSettings.cs

```diff
@@ -53,7 +69,8 @@ public ConnectionManagerSettings(NodeSettings nodeSettings)
 
             try
             {
-                this.AddNode.AddRange(config.GetAll("addnode", this.logger).Select(c => c.ToIPEndPoint(nodeSettings.Network.DefaultPort)));
+                foreach (IPEndPoint addNode in config.GetAll("addnode", this.logger).Select(c => c.ToIPEndPoint(nodeSettings.Network.DefaultPort)))
```

Comment:
How about adding them all via `AddRange`?

Reply:
It seems a bit superfluous adding an overload just for that. A bulk add only happens once on startup typically.